### ISSUE TITLE
Move to modern tekkers?

### ISSUE THREAD

**Issue by larister** [asyncjs/Fishcotheque#3]
e.g. Webpack, ES6 etc

**Comment by orangespaceman** [comment]
I had a think about this, I don't think it's worth it. I like the idea that you can check out the project and run it straight in a browser without needing to touch the console or run a web server. It would be a shame to add extra barriers to entry.

**Comment by larister** [comment]
Yeah I was of the same opinion; although we could just run webpack as a pre-commit hook to update the dist and keep it sans web server - not sure it's worth it though. We probably could broadly upgrade to ES6 if we wanted as we don't really have to support legacy browsers, but I don't think we should rewrite the whole codebase specifically - more allow people to use it if they wish.
